samples/visual_pane/components/visual_pane.js

```diff
@@ -0,0 +1,74 @@
+AFRAME.registerComponent("visual-pane", {
+  schema: {
+    jsonData: {
+      parse: JSON.parse,
+      stringify: JSON.stringify,
+    },
+  },
+
+  multiple: true,
+
+  init: function () {
+    const data = this.data.jsonData;
+    const el = this.el;
+    let imageEl = document.createElement("a-image");
+    imageEl.setAttribute("src", data.imageSrc);
+
+    let rawImageEl = document.createElement("img");
+    rawImageEl.src = data.imageSrc;
+    rawImageEl.onload = function () {
```

Comment:
I think the `.onload` needs to be before adding the `.src` so I would switch it around. Because otherwise the image may load before the onload listener is defined

Reply:
good catch